### PR TITLE
Update build type ids for expected cache miss

### PR DESCRIPTION
**Pull request by blindpirate**
As a followup of portable DSL change on TeamCity.

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test thsi

**Comment by blindpirate** [comment]
@bot-gradle test this

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts [hunk 1]
```diff
@@ -116,13 +117,15 @@ fun isExpectedCompileCacheMiss() =
 // 3. buildScanPerformance test, which doesn't depend on compileAll
     // 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux",
-        "Gradle_Check_Gradleception"
+        "Check_CompileAll",
+        "Check_CompileAll",
+        "Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
```
Comment (wolfs) [inline build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts:122]: It seems like the build type ids of these things have changed to:
- `Enterprise_Master_Component_GradlePlugin_Performance_PerformanceLatestMaster` and
- `Enterprise_Master_Component_GradlePlugin_Performance_PerformanceLatestReleased`

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts [hunk 2]
```diff
@@ -116,13 +117,15 @@ fun isExpectedCompileCacheMiss() =
 // 3. buildScanPerformance test, which doesn't depend on compileAll
     // 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux",
-        "Gradle_Check_Gradleception"
+        "Check_CompileAll",
```
Comment (wolfs) [inline build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts:120]: Why is `CompileAll` there twice?

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts [hunk 3]
```diff
@@ -103,10 +103,11 @@ fun isExpectedAsciidoctorCacheMiss() =
...
     // 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Gradle_Check_BuildDistributions",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux"
+        "Check_CompileAll",
+        "Check_BuildDistributions",
+        "Check_CompileAll",
+        "Check_BuildDistributions",
```
Comment (wolfs) [inline build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts:109]: This is duplicated now.

**Review by wolfs — CHANGES_REQUESTED** [review]

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts [hunk 4]
```diff
@@ -103,10 +103,11 @@ fun isExpectedAsciidoctorCacheMiss() =
 // 3. buildScanPerformance test, which doesn't depend on compileAll
     // 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Gradle_Check_BuildDistributions",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux"
+        "Check_CompileAll",
+        "Check_BuildDistributions",
+        "Check_CompileAll",
```
Comment (blindpirate) [inline build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts:108]:
```suggestion
        "Check_BuildDistributions",
```

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts (hunk 3 above)
Comment (blindpirate) [reply build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts]: Removed.

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts (hunk 1 above)
Comment (blindpirate) [reply build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts]: Thanks! I copied from TeamCity.

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts (hunk 2 above)
Comment (blindpirate) [reply build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts:119]: typo, removed.

**Comment by blindpirate** [comment]
@wolfs PTAL!

File: build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts [hunk 5]
```diff
@@ -99,30 +99,32 @@ fun isMonitoredAsciidoctorTask() = false // No asciidoctor tasks are cacheable f
...
-    // 4. buildScanPerformance test, which doesn't depend on compileAll
+// 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Gradle_Check_BuildDistributions",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux"
+        "Check_CompileAll",
+        "Check_BuildDistributions",
+        "Enterprise_Master_Component_GradlePlugin_Performance_PerformanceLatestMaster",
```
Comment (wolfs) [inline build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts:108]: We should drop the `Enterprise_Master_` prefix here as well.

Comment (blindpirate) [reply build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts]: Done, PATL!

**Review by wolfs — APPROVED** [review]
LGTM!

**Comment by blindpirate** [comment]
Thanks @wolfs !

**Pull request merged**